trying `make gen`:

```
bin/update_crds.sh
Cloning into '/tmp/tmp.OGDbpXAENl'...
remote: Enumerating objects: 3063, done.
remote: Counting objects: 100% (55/55), done.
remote: Compressing objects: 100% (49/49), done.
remote: Total 3063 (delta 7), reused 49 (delta 6), pack-reused 3008 (from 1)
Receiving objects: 100% (3063/3063), 1.43 MiB | 9.95 MiB/s, done.
Resolving deltas: 100% (137/137), done.
remote: Enumerating objects: 90, done.
remote: Counting objects: 100% (58/58), done.
remote: Compressing objects: 100% (44/44), done.
remote: Total 90 (delta 0), reused 42 (delta 0), pack-reused 32 (from 1)
Receiving objects: 100% (90/90), 16.33 KiB | 1.26 MiB/s, done.
remote: Enumerating objects: 384, done.
remote: Counting objects: 100% (208/208), done.
remote: Compressing objects: 100% (177/177), done.
remote: Total 384 (delta 77), reused 80 (delta 30), pack-reused 176 (from 1)
Receiving objects: 100% (384/384), 1.08 MiB | 5.85 MiB/s, done.
Resolving deltas: 100% (97/97), done.
Updating files: 100% (388/388), done.
fatal: detected dubious ownership in repository at '/tmp/tmp.OGDbpXAENl'
To add an exception for this directory, call:

	git config --global --add safe.directory /tmp/tmp.OGDbpXAENl
make[1]: *** [Makefile.core.mk:458: update-crds] Error 128
make: *** [gen] Error 2
```

--- a/manifests/charts/base/files/profile-demo.yaml
+++ b/manifests/charts/base/files/profile-demo.yaml
@@ -21,6 +21,10 @@ meshConfig:
       opentelemetry:
         port: 4317
         service: opentelemetry-collector.observability.svc.cluster.local
+    - name: jaeger
+      opentelemetry:
+        port: 4317
+        service: jaeger-collector.istio-system.svc.cluster.local        
 
 cni:
   resources:

--- a/manifests/charts/default/files/profile-demo.yaml
+++ b/manifests/charts/default/files/profile-demo.yaml
@@ -21,6 +21,10 @@ meshConfig:
       opentelemetry:
         port: 4317
         service: opentelemetry-collector.observability.svc.cluster.local
+    - name: jaeger
+      opentelemetry:
+        port: 4317
+        service: jaeger-collector.istio-system.svc.cluster.local        
 
 cni:
   resources:

--- a/manifests/charts/gateway/files/profile-demo.yaml
+++ b/manifests/charts/gateway/files/profile-demo.yaml
@@ -21,6 +21,10 @@ meshConfig:
       opentelemetry:
         port: 4317
         service: opentelemetry-collector.observability.svc.cluster.local
+    - name: jaeger
+      opentelemetry:
+        port: 4317
+        service: jaeger-collector.istio-system.svc.cluster.local        
 
 cni:
   resources:

--- a/manifests/charts/gateways/istio-egress/files/profile-demo.yaml
+++ b/manifests/charts/gateways/istio-egress/files/profile-demo.yaml
@@ -21,6 +21,10 @@ meshConfig:
       opentelemetry:
         port: 4317
         service: opentelemetry-collector.observability.svc.cluster.local
+    - name: jaeger
+      opentelemetry:
+        port: 4317
+        service: jaeger-collector.istio-system.svc.cluster.local        
 
 cni:
   resources:

--- a/manifests/charts/gateways/istio-ingress/files/profile-demo.yaml
+++ b/manifests/charts/gateways/istio-ingress/files/profile-demo.yaml
@@ -21,6 +21,10 @@ meshConfig:
       opentelemetry:
         port: 4317
         service: opentelemetry-collector.observability.svc.cluster.local
+    - name: jaeger
+      opentelemetry:
+        port: 4317
+        service: jaeger-collector.istio-system.svc.cluster.local        
 
 cni:
   resources:

--- a/manifests/charts/istio-cni/files/profile-demo.yaml
+++ b/manifests/charts/istio-cni/files/profile-demo.yaml
@@ -21,6 +21,10 @@ meshConfig:
       opentelemetry:
         port: 4317
         service: opentelemetry-collector.observability.svc.cluster.local
+    - name: jaeger
+      opentelemetry:
+        port: 4317
+        service: jaeger-collector.istio-system.svc.cluster.local        
 
 cni:
   resources:

--- a/manifests/charts/istio-control/istio-discovery/files/profile-demo.yaml
+++ b/manifests/charts/istio-control/istio-discovery/files/profile-demo.yaml
@@ -21,6 +21,10 @@ meshConfig:
       opentelemetry:
         port: 4317
         service: opentelemetry-collector.observability.svc.cluster.local
+    - name: jaeger
+      opentelemetry:
+        port: 4317
+        service: jaeger-collector.istio-system.svc.cluster.local        
 
 cni:
   resources:

--- a/manifests/charts/ztunnel/files/profile-demo.yaml
+++ b/manifests/charts/ztunnel/files/profile-demo.yaml
@@ -21,6 +21,10 @@ meshConfig:
       opentelemetry:
         port: 4317
         service: opentelemetry-collector.observability.svc.cluster.local
+    - name: jaeger
+      opentelemetry:
+        port: 4317
+        service: jaeger-collector.istio-system.svc.cluster.local        
 
 cni:
   resources:

